avni-server-api/src/main/java/org/avni/server/web/MetabaseController.java

```diff
@@ -21,5 +26,18 @@ public MetabaseController(MetabaseService metabaseService, UserService userServi
     public void setupMetabase() {
         accessControlService.checkPrivilege(PrivilegeType.EditOrganisationConfiguration);
         metabaseService.setupMetabase();
-}
+    }
+
+    @PostMapping("/create-questions")
+    public void createQuestions() throws Exception{
+        databaseService.createQuestionsForSubjectTypes();
+        databaseService.createQuestionsForProgramsAndEncounters();
```

Comment:
databaseService call to createQuestionsForSubjectGeneralEncounterTypes() is missing..